packages/deno/src/generators/application/schema.json

```diff
@@ -56,6 +56,7 @@
     "framework": {
       "type": "string",
       "description": "Generate an app with a framework already setup",
+      "x-prompt": "What framework would you like to use?",
       "enum": ["oak", "none"]
```

Comment:
Would `oak` be the preselected one here, given it's the first? Not into Deno, so I'm wondering whether that's a reasonable choice or whether we should have `none` as the first item

Reply:
Yeah Oak is preselected and is kind of one of the more well known frameworks at this point but might not be what everyone wants so using 'None' might be the 'safer' option for those who just press enter without reading and we default in code to use 'none'. I'll make 'none' default for now and maybe we can switch it up if we support more frameworks down the road.